tests/labhub_test.py

```diff
@@ -88,22 +88,26 @@ def test_create_issue_cmd(self):
         plugins.labhub.GitHubToken.assert_called_with(None)
         plugins.labhub.GitLabPrivateToken.assert_called_with(None)
 
-
         labhub.REPOS = {'repository': self.mock_repo,
                         'repository.github.io': self.mock_repo}
 
         testbot.assertCommand('!new issue repository this is the title\nbo\ndy',
                               'Here you go')
 
         labhub.REPOS['repository'].create_issue.assert_called_once_with(
-            'this is the title', 'bo\ndy\nOpened by @None at [text]()'
+            'this is the title', 'bo\ndy\nOpened on text'
         )
 
-        testbot.assertCommand('!new issue repository.github.io another title\nand body',
-                              'Here you go')
+        msg = Message('!new issue repository.github.io another title\nand body',
+                      'Here you go')
```

Comment:
@jayvdb Invalid command args, it can be like
```
msg = Message('!new issue repository.github.io another title\nand body')
testbot.assertCommand(msg.body, 'Here you go')
```

Reply:
Did you test this?

Reply:
I didn't, I can say that from the errbot documentation, just change constructor and add assert command.